chat-plugins/info.js

```diff
@@ -1082,6 +1088,7 @@ var commands = exports.commands = {
 		} else {
 			return this.sendReplyBox("" + Tools.escapeHTML(target) + " isn't a recognized type or pokemon.");
 		}
+		if (type1 === type2) return this.sendReplyBox("Primary and secondary type cannot be the same.");
```

Comment:
We may just silently ignore the second type changing a line above to
<code>} else if (type1.exists && type2.exists && type1 !== type2) {</code>